platform/macos/godot_content_view.mm

```diff
@@ -882,4 +882,33 @@ - (void)scrollWheel:(NSEvent *)event {
 	}
 }
 
+- (void)rotateWithEvent:(NSEvent *)event {
+	DisplayServerMacOS *ds = (DisplayServerMacOS *)DisplayServer::get_singleton();
+	if (!ds || !ds->has_window(window_id)) {
+		return;
+	}
+
+	DisplayServerMacOS::WindowData &wd = ds->get_window(window_id);
+	ds->update_mouse_pos(wd, [event locationInWindow]);
+
+	Ref<InputEventRotateGesture> ev;
+	ev.instantiate();
+	ds->get_key_modifier_state([event modifierFlags], ev);
+	ev->set_position(wd.mouse_pos);
+	// Convert from degrees to radians and invert direction to match standard rotation direction
+	ev->set_rotation(-[event rotation] * (Math_PI / 180.0));
+
+	Input::get_singleton()->parse_input_event(ev);
+}
+
+- (void)handleGestureEvent:(NSEvent *)event {
+	switch ([event type]) {
+		case NSEventTypeRotate:
+			[self rotateWithEvent:event];
+			break;
+		default:
+			break;
+	}
+}
```

Comment:
What's this part for? There's no `handleGestureEvent` selector in `NSResponder` or `NSView` and it seems like it's never called.

Reply:
Ah, sorry, had experimented quite a bit with the code, this is a remainder, will get rid of it.